usr.bin/procstat/procstat_auxv.c

```diff
@@ -233,26 +248,26 @@ procstat_auxv(struct procstat *procstat, struct kinfo_proc *kipp)
 #endif
 #ifdef AT_ENVV
 		case AT_ENVV:
-			xo_emit("{dw:/%s}{Lw:/%-16s/%s}{:AT_ENVV/%p}\n",
-			    prefix, "AT_ENVV", auxv[i].a_un.a_ptr);
+			xo_emit("{dw:/%s}{Lw:/%-16s/%s}{:AT_ENVV/%#s}\n",
```

Comment:
What is `%#s`?

Reply:
Probably a typo for `%s`.  I suspect that originally this started out as just using `%#p` but then all the invalid attributes required a helper function using strfcap instead.

Reply:
yeah, typo. Lack of format checks (and xolint output being useless in this case since it complains about every single xoemit) is lame.

Reply:
Hmm, would tagging xo_emit as `__printf(1, 2)` work to get format checking I wonder?

Reply:
Unfortunately, you need a parser because things like `{Lw:/%-16s/%s}` are a conditional format.